contents/docs/llm-analytics/start-here.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting started with LLM analytics
+hideAnchor: true
+---
+
+import { QuestLog, QuestLogItem } from 'components/Docs/QuestLog'
+import { IconGraph, IconWarning, IconRewindPlay  } from '@posthog/icons'
+import LLMsInstallationPlatforms from './_snippets/llms-installation-platforms.tsx'
+
+
+<QuestLog firstSpeechBubble="Let's get started!" lastSpeechBubble="Time to integrate LLM analytics!">
+
+<QuestLogItem 
+    title="Capture LLM conversations"
+    subtitle="Required"
+    icon="IconLlmPromptManagement"
+>
+
+LLM analytics gives you x-ray vision into your LLM applications. You can track:
+
+- 🗣️ Every conversation (inputs, outputs, and tokens)
+- 🤖 Model performance (cost, latency and error rates)
+- 🔍 Full traces for when you need to go detective mode
+- 💰 How much each chat/user/organization is costing you
+
+  
+The first step is to install a PostHog SDK to capture conversations, requests, and responses from an LLM provider.
+
+### Platforms
+
+<LLMsInstallationPlatforms />
+
+### Beta 🚧
+
+LLM analytics is currently considered in `beta`. To access it, enable the [feature preview](https://app.posthog.com/settings/user-feature-previews#llm-observability) in your PostHog account.
+
+We are keen to gather as much feedback as possible so if you try this out please let us know. You can email [peter@posthog.com](mailto:peter@posthog.com) and [radu@posthog.com](mailto:radu@posthog.com), send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Aexperiments%3Alow), or use one of our other [support options](/docs/support-options).
+
+<CallToAction type="primary" to="/docs/llm-analytics/installation">
+Install PostHog SDK
+</CallToAction>
+
+</QuestLogItem>
+
+<QuestLogItem 
+  title="Record generations"
```

Comment:
This being a step makes it seem like you have to do something else after installing the SDK.

Also, the term "Record" generations is a bit weird, we don't use record anywhere else other than session replay.